Stellar-contract-meta.x

```diff
@@ -0,0 +1,29 @@
+// Copyright 2022 Stellar Development Foundation and contributors. Licensed
+// under the Apache License, Version 2.0. See the COPYING file at the root
+// of this distribution or at http://www.apache.org/licenses/LICENSE-2.0
+
+// The contract spec XDR is highly experimental, incomplete, and still being
+// iterated on. Breaking changes expected.
+
+% #include "xdr/Stellar-types.h"
+namespace stellar
+{
+
+struct SCMetaV0
+{
+    string key<10>;
+    string val<256>;
```

Comment:
I don't remember where these size limits came from, but they're arbitrary. Maybe 1024? I guess we could even make them unbounded (which is really bounded to u32::MAX), but some reasonable limit would be better.

Reply:
Unbounded might make more sense to give users flexibility. The WASM contract will already be subject to a limit, and users could write whatever they want into the custom section if they wanted to.

Reply:
👍🏻

Reply:
Done